Frontend/src/components/MentorProfile.jsx

```diff
@@ -2,66 +2,178 @@ import {useContext, useEffect, useState} from 'react'
+    'Sales',
+    'Research',
+    'Programming',
+    'Networking',
+    'Graphic Design',
+    'Content Creation',
+    'SEO',
+    'Social Media Management',
+    'UX/UI Design',
+    'Cybersecurity',
+    'Cloud Computing',
+    'Database Management',
+    'Machine Learning',
+    'Artificial Intelligence',
+    'Supply Chain Management',
+    'Quality Assurance',
+    'Product Development',
+    'Business Analysis',
+    'Human Resources',
+    'Training and Development',
+    'Legal Knowledge',
+    'Foreign Languages'
+];
+
+const experienceMappingReverse = {
+    1: '0-2',
+    2: '2-5',
+    3: '5-10',
+    4: '10+',
+    5: '20+',
+  };
 
 function MentorProfile() {
     const { user } = useContext(UserContext)
+    const [selectedSkills, setSelectedSkills] = useState([]);
+    const [dropdownOpen, setDropdownOpen] = useState(false);
+    const [isModalOpen, setIsModalOpen] = useState(false)
     const navigate = useNavigate();
+    const logoApiKey = 'KEY HERE';
+    
+    const fetchLogo = () => {
+        // use api to fetch logo
+    };
+
     const [userData, setUserData] = useState({
         name: "Loading",
-        profileImageUrl: PLACEHOLDER
-    })
+        profileImageUrl: PLACEHOLDER,
+        industry: '',
+        company: '',
+        work_role: '',
+        years_experience: '',
+        school: '',
+        skills: ''
+    });
 
-    useEffect(() => {
+    const fetchMentorData = () => {
         if (user && user.id) {
-            fetch(`http://localhost:3000/users/${user.id}`)
+          fetch(`http://localhost:3000/mentors/${user.id}`)
             .then(response => {
-                if (!response.ok) {
-                    throw new Error(`HTTP status ${response.status}`);
-                }
-                return response.json();
+              if (!response.ok) {
+                throw new Error(`HTTP status ${response.status}`);
+              }
+              return response.json();
             })
             .then(data => {
-                setUserData({
-                    name: data.name,
-                    profileImageUrl: data.profileImageUrl || PLACEHOLDER
-                });
+              setUserData({
+                name: user.name,
+                profileImageUrl: user.profileImageUrl || PLACEHOLDER,
+                industry: data.mentor.industry,
+                company: data.mentor.company,
+                work_role: data.mentor.work_role,
+                years_experience: experienceMappingReverse[data.mentor.years_experience] || '',
+                school: data.mentor.school,
+                skills: data.mentor.skills
+              });
             })
             .catch(error => {
-                console.error('Error fetching user data:', error);
-                setUserData({
-                    name: 'Failed to load user data',
-                    profileImageUrl: PLACEHOLDER
-                });
+              console.error('Error fetching mentor data:', error);
```

Comment:
remove console log (or ad a TODO to remove it)

Reply:
We talked yesterday and aligned on not having console log/error on the front. For backend code I think it's ok to have them because it sort of mimics the server side logging we have. 

There are also some discussion online on whether console.error is helpful. In FB production code we will have logging features for you to properly log the errors from both front end and back end.

We can discuss this further but overall I like the Dominic took the advice and no more console.log in this PR so far.

Reply:
ya makse sense, I didnt actually notice this was an error when I read it. lol. I just saw console and assumed it was a console log